phylanx/execution_tree/primitives/base_primitive.hpp

```diff
@@ -283,6 +284,16 @@ namespace phylanx { namespace execution_tree { namespace primitives
             return 0;
         }
 
+        const std::int64_t get_count()
```

Comment:
That's not what I meant, `const std::int64_t get_count()` means that the returned value is `const`, which is meaningless as it is an rvalue anyways. WhatI meant was that this member function should be callable for `const` objects: `std::int64_t get_count() const`.

Reply:
Fixed.